src/transaction/log_record.hpp

```diff
@@ -152,11 +152,11 @@ enum log_rectype
                                  * it contains transaction user info, DDL statement, undo lsa, redo lsa for DML,
                                  * or undo images that never retrieved from the log. */
 
-  LOG_SCHEMA_MODIFICATION_LOCK, /* Log when SCH_M_LOCK is acquired, and this is for PTS replication to know
-                                 * when to acquire a SCH_M_LOCK for replicating DDL modification, and
-                                 * which table to be locked.
-                                 * PTS needs to block the read transactions which try to access the same class
+  LOG_LOCKED_OBJECT,            /* Log when the lock is acquired for DDL operation, and this is for PTS replicator
```

Comment:
I think the name should include more specific context like 
- `LOG_LOCK_NON_MVCC_OBJECT`
- or `LOG_LOCK_OBJECT_ON_PTS`.

The first one describes when the record is created and the second one is the purpose of this lock. I prefer the second. Or your suggestion.

Reply:
How about LOG_LOCK_INFO_FOR_DDL_REPL?

Reply:
Hi! 
Hi! Thank you for your opinions. I tried to reference the logs used exclusively in PTS, but it wasn't helpful.
- Replication logs (e.g., LOG_START_ATOMIC_REPL, LOG_END_ATOMIC_REPL)
- Logs conveying information for PTS (e.g., LOG_ASSIGNED_MVCCID)

Therefore, based on the suggestions you provided, I aimed to incorporate details about what information the logs carry and what actions they perform. 
- lock information obtained during DDL execution and 
- convey the meaning of replication.

 ** `LOG_REPL_DDL_LOCK_INFO` **
Is it acceptable to proceed in this manner?